cmd/iam/create.go

```diff
@@ -53,7 +53,7 @@ func init() {
 		"", "Service Account Name")
 	CallCmd.Flags().BoolVarP(&generateName, "gen", "g",
 		false, "Generate account name")
-	CallCmd.Flags().StringVarP(&roleType, "role", "r",
+	CallCmd.Flags().StringVarP(&roleType, "role", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `role` is breaking change right?